src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        List list = new List();
+
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+
+        String userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+            if (userInput.equals("list")) {
+                list.printList();
+            }
+
+            else if(userInput.startsWith("done")){   //cannot use command.equals since user is entering more than just "done"
+                list.isComplete(userInput);
+            }
+
+
+            else if (userInput.startsWith("deadline")) {
+                list.addDeadline(getDeadlineDescription(userInput),getStringAfterByAt(userInput));
+            }
+
+
+            else if (userInput.startsWith("event")) {
+                list.addEvent(getEventDescription(userInput),getStringAfterByAt(userInput));
+            }
+
+            else if (userInput.startsWith("todo")) {
+                list.addTodo(userInput.substring(5));
+            }
+
+            else {
+                list.addTask(userInput);
+                System.out.println("added: " + userInput + "\n");
+            }
+
+            userInput = in.nextLine(); //read next user input
+        }
```

Comment:
You may wish to add some short comment for each if, else ifs and else so that others do not need to read what the 'else if' is suppose to check. Also, the else ifs and else should be on the same line as the } from the previous else statement.